doc: matter: add Thread SSED info to window cover (#7947)
nrfconnect/sdk-nrf: samples/matter/window_covering/README.rst

See `User interface`_ for information about how to switch the movement modes.

Synchronized Sleepy End Device activity
=======================================

> Because it is working as a Thread Synchronized Sleepy End Device, the window covering device based on this sample is synchronized with its parent router and uses the radio only at scheduled intervals.

Reviewer: ```suggestion
Because it is working as a Thread Synchronized Sleepy End Device, the window covering device based on this sample is synchronized with its parent router and uses the radio only at scheduled intervals, by utilizing Thread 1.2 Coordinated Sampled Listening (CSL) feature.
```

During those intervals, the device waits for the router to send it any data related to the window cover movement.

> The SSED does not actively communication to the router and goes into the idle mode in-between the scheduled activity periods.

Reviewer: ```suggestion
The SSED does not actively communicate to the router and goes into the idle mode in-between the scheduled activity periods.
```

> The SSED does not actively communication to the router and goes into the idle mode in-between the scheduled activity periods.

Reviewer 2: ```suggestion
Unlike the regular SED device, the SSED does not actively communicate with the router (by polling) and goes into the idle mode in-between the scheduled activity periods.
```

> Overall, this further reduces energy consumption compared with a standard Thread SED.

Reviewer: I think we could add additional information regarding synchronization maintenance, e.g. "The SSED device requires sending packets once for a time to keep synchronization with the router. In case of the absence of application related traffic for a longer period of time, the SSED sends data poll request packet to synchronize with the parent."
